p2p/conn/secret_connection.go

```diff
@@ -14,8 +14,8 @@ import (
 	"time"
 
 	gogotypes "github.com/gogo/protobuf/types"
-	"github.com/gtank/merlin"
 	pool "github.com/libp2p/go-buffer-pool"
+	"github.com/oasisprotocol/curve25519-voi/primitives/merlin"
```

Comment:
This is more performant or?

Reply:
The code's nicer (IMO), more full featured and it removes 2 dependencies.  IIRC my strobe implementation is more performant for "sensible" message sizes, but the difference shouldn't be significant or noticeable for something like this.